modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -443,6 +443,10 @@ export interface SendManyOptions extends PrebuildAndSignTransactionOptions {
   gasLimit?: number;
 }
 
+export interface FetchCrossChainUTXOsOptions {
+  sourceChain?: string;
```

Comment:
Are `P` and `C` the only options for this param? If so, could the type be `sourceChain?: 'P' | 'C';`?

Reply:
yes